### ISSUE TITLE
Remove lightblue-core-hystrix as a dependency for the client

### ISSUE THREAD

**Issue by dcrissman** [lightblue-platform/lightblue-client#273]
This ticket is split off from #267 from the hystrix elements.

The LightblueHystrixClient has a dependency on lightblue-core in order to use the ServoGraphiteSetup utiliy class. To resolve this issue, we either need to split the class in core into it's own library (that supports jdk7) or copy it into lightblue-client.

**Comment by jewzaam** [comment]
Long term we are talking about taking hystrix out of lightblue core and leaving it in client but we're not there yet.  This is related to https://github.com/lightblue-platform/lightblue-core/issues/312.

As you say, break it out or copy it to client...  I think it would be simpler and align with long term goals to copy it to client.  But I don't know when we'll be able to remove the dependency in core.  But, that hystrix stuff in core doesn't change much, so the risk of a copy and getting stale is low.

**Comment by dcrissman** [comment]
I am just going to split into it's own library. That will fix all these issues at the same time.

Created https://github.com/lightblue-platform/lightblue-graphite-publisher

**Comment by jewzaam** [comment]
Is this for hystrix?  Graphite != Hystrix in my head.  We're using hystrix
for the metrics colleciton but would still publish to graphite in the
future when it's removed.  It would require changes to remove hystrix and
do something else.

**Comment by dcrissman** [comment]
That is kind of why I chose the name, although I am not committed to it. I wanted to pick something that was generic enough to not tie us to a specific means of collecting metrics.

**Comment by dcrissman** [comment]
That said, I didn't change the package names for backwards compatibility... so I suppose that doesn't align with the name. Suggestions?

**Comment by bserdar** [comment]
Deprecate them all.

**Comment by dcrissman** [comment]
I have continued to think about this and I have changed my mind. Forget lightblue-graphite-publisher and just go back to the copy/paste idea that @jewzaam was promoting. Specifically, by adding the classes to lightblue-client-hystrix it keeps everything contained in a single place. It is likely that everything that is going to use hystrix to communicate with lightblue via java, is likely going to just use the lightblue-client-hystrix library anyway.

lightblue-applications, lightblue-camel, and all our internal applications are using the client. The more I think about it, the less a new project makes sense to me.
